### PR TITLE
🔧 Refactor code base to be modular for additional entities

### PR DESCRIPTION
**Pull request by znatty22**

### CODE REVIEW

File: dataservice/api/__init__.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
+from flask import Blueprint
 from flask_restplus import Api
 from .person import person_api
 
-api = Api(title='Kids First Data Service',
+api_v1 = Blueprint('api', __name__, url_prefix='/api/v1')
```
Comment (dankolbman) [inline dataservice/api/__init__.py]: We'll probably host under `api.kids-first.io`. We can use `/v1` for versioning, but let's drop the `/api` for now.

File: dataservice/api/common/model.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 from sqlalchemy.ext.declarative import declared_attr
 
 from datetime import datetime
-from . import db
+from dataservice import db
```
Comment (dankolbman) [inline dataservice/api/common/model.py]: why the change from relative to absolute imports? Should probably stay consistent throughout the code base.

File: dataservice/commands.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Click commands."""
+
+import os
+
+import click
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+PROJECT_ROOT = os.path.join(HERE, os.pardir)
+TEST_PATH = os.path.join(PROJECT_ROOT, 'tests')
```
Comment (dankolbman) [inline dataservice/commands.py]: Are these used anywhere?

**Comment by dankolbman** [comment]
We can also remove flask-script from `requirements.txt` if it's being removed.

File: dataservice/commands.py (hunk 3 above)
Comment (znatty22) [reply dataservice/commands.py]: Nope, I'll remove them

File: dataservice/api/common/model.py (hunk 2 above)
Comment (znatty22) [reply dataservice/api/common/model.py]: Ok as just discussed, we're going with absolute imports. Will make the necessary changes

**Review by dankolbman — APPROVED** [review]
lgtm
